src/index.ts

```diff
@@ -43,7 +43,7 @@ try {
   appPackageJson = fs.readJSONSync(resolveApp('package.json'));
 } catch (e) {}
 
-const stat = util.promisify(fs.stat);
+const stat = util.promisify(fs.stat as any);
```

Comment:
Strangely, `util.promisify` is not inferring the callback type of `fs.stat` correctly (`{}` instead of `fs.Stats`)

Reply:
this isn't needed as fs-extra already promisifies everything